code_challenges/code_wars/your_order_please.py

```diff
@@ -0,0 +1,12 @@
+# https://www.codewars.com/kata/55c45be3b2079eccff00010f
+
+
+def sort_order(order):
+    numbers = {1, 2, 3, 4, 5, 6, 7, 8, 9}
```

Comment:
No blocking observation - no need to change. These are only used as strings, so could be strings here like `numbers = {"1", "2", "3", "4", "5", "6", "7", "8", "9"}`